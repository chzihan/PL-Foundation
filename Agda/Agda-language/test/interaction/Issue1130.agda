-- {-# OPTIONS -v tc.with:40 #-}

id : (A : Set) → A → A
id A = {!id′!}
 -- C-c C-h produces:       id′ : ∀ {A} → A
 -- when it should produce: id′ : ∀ {A} → A → A

f : (A : Set) (B : A → Set) (a : A) → B a
f A B a = {!g A a!}
  -- Before: ∀ {A} {B : A → Set} A₁ (a : A₁) → B a
  -- After:  ∀ A (a : A) {B : A → Set} → B a
