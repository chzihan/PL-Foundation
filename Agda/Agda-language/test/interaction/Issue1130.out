(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : A → A ?1 : B a " nil)
((last . 1) . (agda2-goals-action '(0 1)))
(agda2-status-action "")
(agda2-info-action-and-copy "*Helper function*" "id' : ∀ {A} → A → A " nil)
((last . 1) . (agda2-goals-action '(0 1)))
(agda2-status-action "")
(agda2-info-action-and-copy "*Helper function*" "g : ∀ A (a : A) {B : A → Set} → B a " nil)
((last . 1) . (agda2-goals-action '(0 1)))
